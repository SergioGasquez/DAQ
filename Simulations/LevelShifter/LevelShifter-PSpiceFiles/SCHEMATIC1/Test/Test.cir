** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\LevelShifter\levelshifter-pspicefiles\schematic1\test.sim ] 

** Creating circuit file "Test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\sergi\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.DC LIN V_V6 0 5 0.01 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 


.END
