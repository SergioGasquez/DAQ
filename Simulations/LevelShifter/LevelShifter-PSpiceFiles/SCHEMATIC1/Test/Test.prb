[DISPLAYS]
BEGIN DISPLAY LAST SESSION
ANALYSIS DC_SWEEP
SYMBOL ALWAYS
TRACECOLORSCHEME NORMAL
BEGIN CURSORS
CURSORNUMDIGITS 5
BEGIN CURSOR 1
CURSOR_X 4.98999999999994
CURSOR_Y 4.49181957722057
ANAPLOT 1
ANATRACE 0
END CURSOR 1
BEGIN CURSOR 2
CURSOR_X 0
CURSOR_Y 0.0397666035622113
ANAPLOT 1
ANATRACE 0
END CURSOR 2
END CURSORS
BEGIN ANAPLOT 1
ACTIVE
XBASE
BEGIN XAXIS
XAXISUSERNAME 0 (null)
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
END XAXIS
BEGIN YAXIS 1
YAXISSIDE LEFT
ACTIVE
RANGEFLAG AUTO
TYPE LINEAR
UNIT V
BEGIN GRIDS
AUTOMATIC
MAJORNUMBERS
MAJORSTYLE LINES
MAJORPATTERN SOLID
MINORSTYLE LINES
MINORPATTERN DOT
END GRIDS
BEGIN TRACE V(V6:+)
MARKERID 7
END TRACE V(V6:+)
BEGIN TRACE V(Vo)
MARKERID 8
END TRACE V(Vo)
END YAXIS 1
END ANAPLOT 1
END DISPLAY LAST SESSION
