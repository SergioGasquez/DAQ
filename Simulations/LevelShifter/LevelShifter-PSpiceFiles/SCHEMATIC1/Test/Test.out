
**** 04/05/18 19:27:03 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\LevelShifter\levelshifter-pspicefiles\schematic1\te


 ****     CIRCUIT DESCRIPTION


******************************************************************************




** Creating circuit file "Test.cir" 
** WARNING: THIS AUTOMATICALLY GENERATED FILE MAY BE OVERWRITTEN BY SUBSEQUENT SIMULATIONS

*Libraries: 
* Profile Libraries :
* Local Libraries :
* From [PSPICE NETLIST] section of C:\Users\sergi\AppData\Roaming\SPB_Data\cdssetup\OrCAD_PSpice\17.2.0\PSpice.ini file:
.lib "nomd.lib" 

*Analysis directives: 
.DC LIN V_V6 0 5 0.01 
.OPTIONS ADVCONV
.PROBE64 V(alias(*)) I(alias(*)) W(alias(*)) D(alias(*)) NOISE(alias(*)) 
.INC "..\SCHEMATIC1.net" 



**** INCLUDING SCHEMATIC1.net ****
* source LEVELSHIFTER
X_U2B         N02288 N02380 N02284 N02424 VO LM358
V_V6         N02346 GND 5Vdc
R_R2         N02288 GND  50k TC=0,0 
R_R3         N024000 N02288  200k TC=0,0 
V_V3         N02424 GND -15Vdc
R_R1         N02346 N02288  100k TC=0,0 
R_R5         GND N02380  47k TC=0,0 
R_R4         N02380 VO  100k TC=0,0 
V_V2         N024000 GND -5Vdc
V_V4         N02284 GND 15Vdc

**** RESUMING Test.cir ****
.END

**** 04/05/18 19:27:03 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\LevelShifter\levelshifter-pspicefiles\schematic1\te


 ****     Diode MODEL PARAMETERS


******************************************************************************




               X_U2B.x_lm358.dx 
          IS  800.000000E-18 
          RS    1            


               X_U2B.x_lm358.dy 
          IS  800.000000E-18 
          RS    1.000000E-03 
         CJO   10.000000E-12 


**** 04/05/18 19:27:03 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\LevelShifter\levelshifter-pspicefiles\schematic1\te


 ****     BJT MODEL PARAMETERS


******************************************************************************




               X_U2B.x_lm358.qx 
               PNP             
       LEVEL    1            
          IS  800.000000E-18 
          BF  166.7          
          NF    1            
          BR    1            
          NR    1            
         ISS    0            
          RE    0            
          RC    0            
         CJE    0            
         VJE     .75         
         CJC    0            
         VJC     .75         
         MJC     .33         
        XCJC    1            
         CJS    0            
         VJS     .75         
          KF    0            
          AF    1            
          CN    2.2          
           D     .52         


**** 04/05/18 19:27:03 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\LevelShifter\levelshifter-pspicefiles\schematic1\te


 ****     Current Controlled Switch MODEL PARAMETERS


******************************************************************************




               X_U2B.x_lm358.sw2 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 


               X_U2B.x_lm358.sw1 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U2B.x_lm358.sw3 
         RON    1            
        ROFF   10.000000E+06 
         ION   -1.000000E-06 
        IOFF    0            


               X_U2B.x_lm358.sw4 
         RON    1            
        ROFF   10.000000E+06 
         ION    0            
        IOFF   -1.000000E-06 



          JOB CONCLUDED

**** 04/05/18 19:27:03 ******* PSpice Lite (March 2016) ******* ID# 10813 ****

 ** Profile: "SCHEMATIC1-Test"  [ C:\Users\sergi\Documents\GitHub\DAQ\Simulations\LevelShifter\levelshifter-pspicefiles\schematic1\te


 ****     JOB STATISTICS SUMMARY


******************************************************************************



  Total job time (using Solver 1)   =         .05
